src/services/web3Service.ts

```diff
@@ -179,14 +182,16 @@ export default class Web3Service {
       this.providerService,
       this.safeService
     );
+    this.connextService = new ConnextService(this.walletService, 137); // hardcoding polygon ID
```

Comment:
how would this work if its not just polygon? do we need a connextService for each chain?

Reply:
Nope, we are only demonstrating arbitrum to polygon transfer as of now, this will be changed to dynamic input later after discussing the UX

Reply:
We should remove the second parameter right? Since its going to depend on when you create the position